spinnaker/await_restart_spinnaker/await_restart_spinnaker.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+function print_usage() {
+  cat <<EOF
+Command
+  $0 
+
+Arguments
+  --timeout|-t   : Timeout in seconds, defaulted to 120
+EOF
+}
+
+# Set defaults
+timeout=120
+
+while [[ $# > 0 ]]
+do
+  key="$1"
+  shift
+  case $key in
+    --timeout|-t)
+      timeout="$1"
+      shift
+      ;;
+    --help|-help|-h)
+      print_usage
+      exit 13
+      ;;
+    *)
+      echo "ERROR: Unknown argument '$key'" 1>&2
+      exit -1
+  esac
+done
+
+sudo restart spinnaker
+
+# Wait for the following Spinnaker services to be ready:
+# Gate - port 8084
+# Clouddriver - port 7002
+# Front50 - port 8080
+# Orca - port 8083
+count=0
+while !(nc -z localhost 8080) || !(nc -z localhost 8084) || !(nc -z localhost 7002) || !(nc -z localhost 8083); do
```

Comment:
There's a very small chance the user changes their ports and tries to use this script and fails. It might be possible to dynamically check which port maps to which service, but I don't think the effort is worth it at this point